res/css/structures/_MatrixChat.scss

```diff
@@ -76,14 +76,28 @@ limitations under the License.
     height: 100%;
 }
 
+// We'd like to remove this, but this makes matrixchat's resizehandle's
+// negative margin & greater than its positive padding. If it's the same
+// or less, Safari gets confused about overflows somehow and
+// https://github.com/vector-im/element-web/issues/19863 happens.
+.mx_MatrixChat > .mx_ResizeHandle.mx_ResizeHandle_horizontal {
+    margin: 0 calc(-6px - $container-gap-width / 2) 0 calc(-6px + $container-gap-width / 2);
```

Comment:
Placing it truly in the centre of the gap seems to require:

```suggestion
    margin: 0 calc(-5.5px - $container-gap-width / 2) 0 calc(-6.5px + $container-gap-width / 2);
```

which relates to your 0.5px question yesterday, which may or may not be okay for non-retina screens, and I don't have one at hand to test at the moment...

Could we change something else to get back to integer units and still be in the centre? If we can't think of anything, I wonder if this fractional positioning could be okay...

Reply:
I am not sure if this is possible to get perfectly centered. I tried lots of values and they all seemed to be slightly off...

Reply:
Okay I tried again on chromium. It seems that there you are absolutely right. On Firefox this seems to behave slightly different if you use pinch to zoom or ctrl+mousewheel zooming. Pinch to zoom almost shows no difference between 5.5px  and 6px.
Just for the record I add a bunch of screenshots. But basically 5.5 & 6.5 are the better values. Dont know what FF is doing.

browser scrolling
![Screenshot from 2021-12-02 16-55-10](https://user-images.githubusercontent.com/16718859/144457435-6d5cb8d6-1d0a-4620-9e81-e3e4de964e8e.png)
![Screenshot from 2021-12-02 16-54-30](https://user-images.githubusercontent.com/16718859/144457422-7902b44e-be66-4cd1-8891-ad6934693271.png)
![Screenshot from 2021-12-02 16-54-51](https://user-images.githubusercontent.com/16718859/144457431-cca12373-0a68-47b2-b600-f1e40ed4d75f.png)

Pinch To zoom:
![Screenshot from 2021-12-02 16-57-17](https://user-images.githubusercontent.com/16718859/144457442-78cedb31-0cdc-4924-ad0b-cde7304c5401.png)
![Screenshot from 2021-12-02 16-58-27](https://user-images.githubusercontent.com/16718859/144457445-a39a2b0e-a4a5-43dc-96a6-5e6e97764309.png)